users/users_test.go

```diff
@@ -28,5 +29,59 @@ func TestAddLinksCard(t *testing.T) {
 }
 
 func TestMaskCC(t *testing.T) {
+	test1 := "1234567890"
+	c := Card{LongNum: test1}
+	c.MaskCC()
+	test1comp := "******7890"
+	if c.LongNum != test1comp {
+		t.Errorf("Expected matching CC number %v received %v", test1comp, test1)
+	}
+}
+
+func TestNew(t *testing.T) {
+	u := New()
+	if len(u.Addresses) != 0 && len(u.Cards) != 0 {
```

Comment:
I would advise you to split these. If the requirements change tomorrow you remove either test and it'll look cleaner, imo, but take it with a grain of salt.